jsonschema-module-jakarta-validation/src/test/java/com/github/victools/jsonschema/module/jakarta/validation/IntegrationTest.java

```diff
@@ -134,6 +136,14 @@ static class TestClass {
         @DecimalMin(value = "0", inclusive = false)
         @DecimalMax(value = "1", inclusive = false)
         public double exclusiveRangeDouble;
+
+        @AssertTrue
+        public boolean trueBoolean;
+        @AssertFalse
+        public boolean falseBoolean;
+        @AssertTrue
+        @AssertFalse
+        public Object allEnumResolvedObject;
```

Comment:
**question**: Isn't this contradictory? How can something be both `true` and `false` simultaneously?
I'd argue this is an invalid use case and we don't need to support it.

We could avoid some complexity if we didn't allow both values to be returned.
For the scenario where you have an `Object` that you want to treat as `boolean`, we should just set it to `"type": "boolean"` instead – but not based on these annotations.
Is this a relevant use case for you, i.e., do we need to offer a way to indicate this type override or is it okay to just remove that part again here?

Reply:
Neither of these is actually relevant to my use case.

I just implemented it that way cause I was thinking of a solution for if a user added both `@AssertTrue` and `@AssertFalse`. But I don't feel particularly strongly about what I did, so I'll just make these annotations mutually  exclusive as you suggested.

Reply:
Fixed the tests for not supporting having both `@AssertTrue` and `@AssertFalse`.